### DISCUSSION TITLE
Can't update or create new records Hosted AWS RDS/Postgresql

### DISCUSSION THREAD

**Discussion by gpalomar12** [Budibase/budibase#12976] [Help] [answered]
I've created and linked Postgresql on both AWS RDS and Postgres and configured PGADMIN with the proper credentials to access AWS RDS instance.  After doing that I was able to setup Budibase to connect through my connection string and it pulls the data with no problems.  I then created 3 tables in the designer, and tried add a populating a new row, when I save it nothing happens and I get an undefined column error prompt.  I can see the data fine, but can't seem to modify anything.  Any ideas?

**Comment by ConorWebb96** [comment]
@gpalomar12,

You will likely need to create the tables initially on the postgres end. Then sync these in. I don't believe creating tables in the design view adds `primary keys`. These are needed to run external tables in Budibase. So make sure you at least create these first before you start building everything else in Budibase. I strongly suspect that there are no primary keys created and this is likely the reason you aren't able to save/create/update anything.

I hope this helps.

**Reply by ConorWebb96** [reply]
If this isn't the case can you provide some screenshots of your connection? (Each of the of the tabs that are shown in the below image.)
<img width="831" alt="Screenshot 2024-02-07 at 08 19 47" src="https://github.com/Budibase/budibase/assets/126772285/30d59126-deb8-49e9-b2aa-96c1c2e1bb36">

Can you also provide your sql schema so, we can see what the structure of these tables look like. 

- What version of Budibase are you running?
- Are you cloud or self-hosted?

Thanks in advance.

**Reply by gpalomar12** [reply, marked as answer]
After looking through the logs in PGAdmin, I realized that I had some triggers created that conflicted with some of the constraints.  I disabled them and it appeared to resolve the issue.  I am running a cloud-based database.
